scripts/benchmark_db/bench.rb

```diff
@@ -58,7 +58,7 @@ def snapshot_dir
 def db_dir
   config = default_config
   data_dir = config.dig('client', 'data_dir')
-  db = config.key?('rocks_db') ? 'rocksdb' : 'paritydb'
+  db = config.key?('rocks_db') ? 'rocksdb' : config.key?('paritydb') ? 'paritydb' : 'lmdb'
```

Comment:
Hmm, well, this won't work because we never serialize a `paritydb` section in config file. It's not a big issue since you won't need to splice into the default config a custom lmdb config.

Reply:
We lack right now identifying in the forest version string what is the db backend.
The simplest thing to do would be to add `+lmdb` or `+paritydb` to Forest version string.

Reply:
Extracting path from `forest-cli db stats` should work.